Procfile

```diff
@@ -5,7 +5,9 @@ worker_buildapi_pending: newrelic-admin run-program celery -A treeherder worker
 worker_buildapi_running: newrelic-admin run-program celery -A treeherder worker -Q buildapi_running --maxtasksperchild=20 --concurrency=5
 worker_buildapi_4hr: newrelic-admin run-program celery -A treeherder worker -Q buildapi_4hr --maxtasksperchild=20 --concurrency=1
 worker_store_pulse_jobs: newrelic-admin run-program celery -A treeherder worker -Q store_pulse_jobs --maxtasksperchild=20 --concurrency=1
+worker_store_pulse_resultsets: newrelic-admin run-program celery -A treeherder worker -Q store_pulse_resultsets --maxtasksperchild=20 --concurrency=1
```

Comment:
I know the `worker_store_pulse_jobs` entry also has a concurrency of 1, but I wonder if one or both can actually handle it being higher? So long as the RAM usage doesn't exceed the 512MG of the P1 dyno (and of course that CPU usage isn't so high that concurrency isn't gaining anything), then we could bump this up.

The current memory usage seems quite low:
https://dashboard.heroku.com/apps/treeherder-stage/metrics/worker_store_pulse_jobs